README.md

```diff
@@ -59,14 +59,45 @@ We use `yarn` in this project and commit the `yarn.lock` file.
    # setup git hooks
    npx simple-git-hooks
    ```
-1. Follow the getting started guides in [`/packages/api`](/packages/api#getting-started) and [`/packages/website`](/packages/website#getting-started).
-1. Run locally by following the instructions below.
+2. Setup your local environment with a `.env` file. See [intructions](#local-environment-configuration).
+3. Follow the getting started guides in [`/packages/api`](/packages/api#usage) and [`/packages/website`](/packages/website#usage).
+4. Run locally by starting the following processes.
+   1. API server (`yarn dev` in `/packages/api`).
+   2. Web server (`yarn dev` in `/packages/website`).
+
+The site should now be available at http://localhost:4000
 
 ## Local environment configuration
 
 In the root folder create a `.env` file with the following:
 
 ```ini
+
+# Wrangler overrides
+DEBUG=true
+
+# API Secrets
+SALT=secret
+MAILCHIMP_API_KEY=secret
+METAPLEX_AUTH_TOKEN=secret
+MAGIC_SECRET_KEY=secret # needs to be real so create a personal magic.link account or use the staging key
+LOGTAIL_TOKEN=secret
+
+## API Sentry
+SENTRY_DSN=https://000000@0000000.ingest.sentry.io/00000
+SENTRY_TOKEN=secret
+SENTRY_UPLOAD=false
+
+## API Postgrest
+DATABASE_URL=http://localhost:3000
+DATABASE_TOKEN=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJzdXBhYmFzZSIsImlhdCI6MTYwMzk2ODgzNCwiZXhwIjoyNTUwNjUzNjM0LCJyb2xlIjoic2VydmljZV9yb2xlIn0.necIJaiP7X2T2QjGeV-FhpkizcNTX8HjDDBAxpgQTEI
+
+# Postgres Database
+DATABASE_CONNECTION=postgresql://postgres:postgres@localhost:5432/postgres
+
+# Vars below this line are optional or can have fake values
```

Comment:
Can't sentry be optional as well?

Reply:
No, it's for miniflare the global needs to be set to something

Reply:
no, this is need for miniflare the global var needs to be set even if its a fake value.